tests/unitTests/src/unitTestsAuth.ts

```diff
@@ -461,7 +470,9 @@ export function runAuthTests() {
 
     const appConfig = new AppConfig(['store_write'], 'http://localhost:3000')
     const blockstack = new UserSession({ appConfig })
-    blockstack.store.getSessionData().transitKey = transitPrivateKey
+
+    const sessionData = await blockstack.store.getSessionData()
+    sessionData.transitKey = transitPrivateKey
 
     blockstack.handlePendingSignIn(authResponse)
```

Comment:
This may be contributing to the failing tests, RE:
> Error: test exited without ending

Since this function is now marked `async` I think this `blockstack.handlePendingSignIn` needs to be `await`ed. Also, the `t.plan` call above may be confusing the test lib, try removing it.

Reply:
Same with any other tests that are now marked as `async` -- I think the `t.plan(n)` statements may need to be removed.

Reply:
Yeah, this is difficult, this `tape` library is not giving me anything.
I tried removing all the `t.plan` statements, but nothing 🤔 
Will try again later.

Reply:
Don't worry too much about the tests -- once we get the implementation / API surface down, I can help get the tests working.